homeassistant/components/risco/models.py

```diff
@@ -0,0 +1,15 @@
+"""Models for Risco integration."""
+
+from collections.abc import Callable
+from dataclasses import dataclass, field
+from typing import Any
+
+from pyrisco import RiscoLocal
+
+
+@dataclass
+class LocalData:
+    """A data class for local data passed to the platforms."""
+
+    system: RiscoLocal
+    partition_updates: dict[int, Callable[[], Any]] = field(default_factory=dict)
```

Comment:
I am not sure what the gain of this is without seeing what you're planning to add, so I would be fine with having such change in the action PR

Reply:
It was requested by the code owner in https://github.com/home-assistant/core/pull/139015 to split the refactoring from the new functionality but I agree that this refactoring a rather an artifact of the newly added functionality in the PR mentioned above. What would you advise?